pkg/helm/values/k3s.go

```diff
@@ -136,15 +138,16 @@ isolation:
 	return values, nil
 }
 
-func ParseKubernetesVersionInfo(versionStr string) (*version.Info, error) {
-	if versionStr[0] == 'v' {
-		versionStr = versionStr[1:]
```

Comment:
@pratikjagrut does `--kubernetes-version v1.20` and `--kubernetes-version 1.20` still work?

Reply:
v1.20, but just 1.20 will not work because it is not in a semvar format.

Reply:
@pratikjagrut I think we should allow 1.20 as well, as its easy to do for us and we definitely know what the user wants to do. Helm for example also allows this for chart versions even though they require semver for it as well.

Reply:
I’ve already updated this to allow 1.20 too.

Reply:
Ah didn't see, my bad! I'll merge it!